crates/cxx-qt-gen/src/lib.rs

```diff
@@ -73,45 +78,84 @@ mod tests {
         code
     }
 
+    fn update_expected_file(path: PathBuf, source: &str) {
+        println!("Updating expected file: {:?}", path);
```

Comment:
do we ever see this trace line ?

Reply:
Yes, if the "test" fails, i.e. file access fails because the provided path points to the wrong directory, etc.
Then some of the unwrapping later will make the test fail and output will be printed.